spec/forms/support_interface/country_form_spec.rb

```diff
@@ -192,4 +192,71 @@
       end
     end
   end
+
+  describe "#for_existing_country" do
+    subject(:form) { described_class.for_existing_country(country) }
+
+    let(:country) { create(:country) }
+    let(:eligibility_route) { "standard" }
+    let(:eligibility_enabled) { true }
+    let(:has_regions) { true }
+    let(:region_names) { "Madrid\nBarcelona" }
+    let(:other_information) { "Other information" }
+    let(:status_information) { "Status information" }
+    let(:sanction_information) { "Sanction information" }
+    let(:teaching_qualification_information) do
+      "Teaching qualification information"
+    end
+
+    context do
+      before do
+        create(:region, name: "Madrid", country:)
+        create(:region, name: "Barcelona", country:)
+      end
+
+      it "returns a CountryForm for the country" do
+        expect(subject).to have_attributes(eligibility_enabled: true)
+        expect(subject).to have_attributes(eligibility_route: "standard")
+        expect(subject).to have_attributes(has_regions: true)
+        expect(subject).to have_attributes(region_names: "Madrid\nBarcelona")
+        expect(subject).to have_attributes(
+          other_information: "Other information",
+        )
+        expect(subject).to have_attributes(
+          status_information: "Status information",
+        )
+        expect(subject).to have_attributes(
+          sanction_information: "Sanction information",
+        )
+        expect(subject).to have_attributes(
+          teaching_qualification_information:
+            "Teaching qualification information",
+        )
+      end
```

Comment:
Hi mate, you don't need to have a line for every single check. You can have a single `have_attributes` to check them all. Have a look at how we did it on the `save!` test.